master/buildbot/www/config.py

```diff
@@ -58,6 +58,8 @@ def renderIndex(self, request):
         config['title'] = self.master.config.title
         config['titleURL'] = self.master.config.titleURL
         config['multiMaster'] = self.master.config.multiMaster
+        if "wamp" in self.master.config.protocols:
+            config['wamp'] = self.master.config.protocols['wamp']
```

Comment:
Its to get the router url. Goal is to have the web ui is wamp as an optimised communication channel. REST + SSE does not scale well according to my tests. (Try using several tabs on nine.buildbot.net)